templates/macros/ini_encode_macro.j2

```diff
@@ -0,0 +1,16 @@
+{#
+ # Jinja2 macro which converts Ansible map into no section INI format
+ #}
+
+{%- macro ini_encode(
```

Comment:
This macro does not cover all ini use-cases, for instance `[sections]` (part of ini [specification](https://en.wikipedia.org/wiki/INI_file#Sections)), which are used by `/etc/rstudio/launcher.conf`, e.g.:

```ini
[server]
address=127.0.0.1
port=5559
server-user=rstudio-server
authorization-enabled=1
thread-pool-size=4
enable-debug-logging=1
admin-group=rstudio-server
[cluster]
name=Local
type=Local
exe=/usr/lib/rstudio-server/bin/rstudio-local-launcher
```

I'm also unsure whether RStudio's implementation allows/forces for values to be quoted to include explicit whitespace, i.e.
```
auth-openid-scopes="scope1 scope2"
```

Reply:
I can add a macro with section support when I will be adding handler for launcher.conf file.

Reply:
It supports spaces but doesnt require quotes:
`auth-openid-scopes=scope1 scope2`